pkg/commands/root.go

```diff
@@ -119,8 +119,9 @@ func formatMemory(memBytes uint64) string {
 
 func getDefaultConcurrency() int {
 	if os.Getenv(envHelpRun) == "1" {
-		// Make default concurrency stable: don't depend on machine CPU number, for docs/src/docs/usage/configuration.mdx help generating builds.
-		return 0
+		// Make stable concurrency for README help generating builds.
```

Comment:
I think this change is misleading. Earlier, the script would populate README file, but now it only populates files in `docs/` folder

And changing returning to non zero (non nil value) will print default (8). User may think the default is 8 when seeing the the website docs